### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Soumyajyotig**
sample hello world code

### CODE REVIEW

File: MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import android.view.View;
+import android.widget.Toast;
+
+public class MainActivity extends AppCompatActivity {
+    public void fun1(View view){
+        Toast.makeText(this, "This is a sample App", Toast.LENGTH_SHORT).show();
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```
Comment (abhibt) [inline MainActivity.java]: You need to put a comment here, what this function is doing.
All functions should have a comment above, as to what the function is doing.

File: MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.soumyajyoti.firstapplication;
```
Comment (abhibt) [inline MainActivity.java]: Keep it with professional name, not soumyajyoti.firstapplication.
com.helloworld.firstapp

**Review by abhibt — CHANGES_REQUESTED** [review]
Work on the review comments and then submit the code again.

**Review by Soumyajyotig — COMMENTED** [review]
I have updated the file as you directed. added comments and updated the file name

File: MainActivity.java (hunk 1 above)
Comment (abhibt) [reply MainActivity.java]: The comment should be outside the function, just above the function. Not at the place where you have written.

**Comment by abhibt** [comment]
Try to write junits. Checkin the junits along with code.

**Pull request merged**